examples/demos/rendering.js

```diff
@@ -1,11 +1,14 @@
-import React from 'react'
+import React, { Children, useEffect, useState } from 'react'
 import { Calendar, Views } from 'react-big-calendar'
 import events from '../events'
 
 function Event({ event }) {
   return (
-    <span>
-      <strong>{event.title}</strong>
+    <span style={{ color: 'black' }}>
+      <strong>
+        {event.allDay ? 'ALL' : 'NOT ALL'}
+        {event.title}
```

Comment:
![image](https://user-images.githubusercontent.com/30227910/109777964-d7785d80-7c36-11eb-8250-d9923bd32eda.png)
When render it will show like that -> I think you should put into 2 span tag to can write unit/integration test easier
We can catch event type and title to testing

Reply:
Opps sorry, this is actually a mistakes :v I forgot to remove it